tests/test_main.py

```diff
@@ -18,14 +18,14 @@
 import unittest
```

Comment:
Could you rename this file to test_execute?

Reply:
Got it! I was getting the [errors travis is getting for 3.4](https://travis-ci.org/google/gif-for-cli/jobs/408651285#L538) when I did that locally so I had left it. But now I realize that's [been there](https://travis-ci.org/google/gif-for-cli/jobs/395679764#L539)